core-bundle/src/Resources/config/services.yml

```diff
@@ -458,11 +458,17 @@ services:
             - '@contao.framework'
             - '@logger'
 
-    contao.security.database_token_provider:
-        class: Contao\CoreBundle\Security\Authentication\RememberMe\DatabaseTokenProvider
+    contao.security.expiring_rememberme_services:
+        class: Contao\CoreBundle\Security\Authentication\RememberMe\ExpiringTokenBasedRememberMeServices
```

Comment:
Based on the naming conventions, if the service ID is `expiring_rememberme_services`, the class name has to be `ExpiringRemembermeServices`. And since it is only one service, it should not be service**s**.

Reply:
They are called `Services` in Symfony. And I just used a shorter service name because having the complete name is kinda ugly. But it adds expiration to the token based rememberme services.

Reply:
Ok, then keep the plural. But you will have to fix the method/service name.